build.gradle.kts

```diff
@@ -70,13 +70,15 @@ tasks {
             Files.createDirectories(parent)
         }
 
-        val version = project.version
-        val buildId = System.getenv("GITHUB_RUN_ID") ?: "<local build>"
-        val commitHash = System.getenv("GITHUB_SHA") ?: gitCommitHash()
+        doLast {
```

Comment:
What does the doLast do here?

Reply:
doLast ensures that the code is executed only when the task is executed and not when the buildscript is evaluated.